manifest.yml

```diff
@@ -0,0 +1,11 @@
+name : "coreMQTT"
+version: "1.0.0"
+description: |
+  "Client implementation of the MQTT 3.1.1 specification for embedded devices . \n"
+dependencies:
+ - name: "CMock"
```

Comment:
hmm CMock seems odd. A customer using this library in production wont need CMock. Its only for unit testing